files/build_templates/startup_tsa_tsb.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description= STARTUP TSA-TSB SERVICE
+Requires=updategraph.service database.service
+After=updategraph.service database.service
+ConditionPathExists=!/etc/sonic/chassisdb.conf
+
+[Service]
+Environment="STARTED_BY_TSA_TSB_SERVICE=1"
+ExecStart=/usr/bin/python3 -u /usr/local/bin/startup_tsa_tsb.py  start
+ExecStop=/usr/bin/python3 -u /usr/local/bin/startup_tsa_tsb.py stop
+RemainAfterExit=yes
```

Comment:
what is the behavior of this services in case of config reload and if the swss docker restart ?

Reply:
config reload or swss restart doesn't restart the tsa service